test/integration/servers/graphql-main-server/galoy-pay-testing.spec.ts

```diff
@@ -0,0 +1,274 @@
+import crypto from "crypto"
+
+import { yamlConfig } from "@config/app"
+import { ApolloClient, NormalizedCacheObject } from "@apollo/client/core"
+
+import ME from "./queries/me.gql"
+import USER_LOGIN from "./mutations/user-login.gql"
+import NODE_IDS from "./queries/node-ids.gql"
+import USER_DEFAULT_WALLET_ID from "./queries/user-default-walletid.gql"
+import LN_INVOICE_CREATE_ON_BEHALF_OF from "./mutations/ln-invoice-create-on-behalf-of-recipient.gql"
+import LN_NO_AMOUNT_INVOICE_CREATE_ON_BEHALF_OF from "./mutations/ln-no-amount-invoice-create-on-behalf-of-recipient.gql"
+import PRICE from "./subscriptions/price.gql"
+import LN_INVOICE_PAYMENT_STATUS from "./subscriptions/ln-invoice-payment-status.gql"
+import LN_INVOICE_PAYMENT_SEND from "./mutations/ln-invoice-payment-send.gql"
+
+import {
+  clearAccountLocks,
+  clearLimiters,
+  getDefaultWalletIdByTestUserIndex,
+} from "test/helpers"
+
+import {
+  createApolloClient,
+  getSubscriptionNext,
+  defaultTestClientConfig,
+} from "test/helpers/apollo-client"
+
+import { startServer, killServer } from "test/helpers/integration-server"
+
+jest.setTimeout(60000)
+
+let apolloClient: ApolloClient<NormalizedCacheObject>,
+  disposeClient: () => void,
+  receivingWalletId
+const receivingUsername = "user0"
+const receivingUserIndex = 0
+const { phone, code } = yamlConfig.test_accounts[4]
```

Comment:
feel weird to have `receivingUserIndex = 0` and `yamlConfig.test_accounts[4]`. maybe adding a `senderAccountIndex = 4` would make it a bit more clearer